src/Extension/FluentExtension.php

```diff
@@ -65,6 +65,24 @@ class FluentExtension extends DataExtension
      */
     const TRANSLATE_NONE = 'none';
 
+    /**
+     * Content inheritance - content will be served from the following sources in this order:
+     * current locale
+     */
+    const INHERITANCE_MODE_NONE = 'none';
```

Comment:
I feel like 'none' is a bit obtuse. How about "exact"? (no fallback, must be localised in exact locale)